### PR TITLE
[Merged by Bors] - Change scaling mode to FixedHorizontal

### PR DESCRIPTION
**Pull request by kirusfg**
# Objective

- Fixes the issue with orthographic camera imported from glTF not displaying anything (mentioned in #4005).

## Solution

- This was due to wrong scaling mode being used. This PR simply changes WindowSize scaling mode to FixedHorizontal.

## Important Note

Currently, othographic scale in Blender, three.js, and possibly other software does not translate to Bevy (via glTF) because their developers have [misinterpreted the spec](https://github.com/KhronosGroup/glTF/issues/1663#issuecomment-618194015). The camera parameters have been clarified in glTF 2.0, which was released on October of 2021. In Blender 3.0.1 this issue has **not** been fixed yet. If you are importing orthographic cameras from Blender, you have to divide the scale by 2.

### CODE REVIEW

**Review by superdump — APPROVED** [review]
The changes look good to me, I just don't know if anyone uses gltf for 2D and sprites so the change from `CAMERA_2D` to `CAMERA_3D` could be controversial. It makes sense to me though.

**Review by mockersf — APPROVED** [review]

**Comment by mockersf** [comment]
I don't think it really makes sense to use gltf for 2D...

**Review by mockersf — COMMENTED** [review]
I think we should not specify `left`/`right`/`top`/`bottom` when `scaling_mode` is `ScalingMode::FixedHorizontal`.

File: crates/bevy_gltf/src/loader.rs [hunk 1]
```diff
@@ -612,14 +613,19 @@ fn load_node(
                     bottom: -ymag,
                     far: orthographic.zfar(),
                     near: orthographic.znear(),
+                    scaling_mode: ScalingMode::FixedHorizontal,
+                    scale: xmag,
```
Comment (mockersf) [inline crates/bevy_gltf/src/loader.rs:617]:
```suggestion
                    scale: xmag / 2.0,
```

doing the math from https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#orthographic-projection, it looks like this need to be divided by 2

Comment (kirusfg) [reply crates/bevy_gltf/src/loader.rs]: Amended the suggested changes

File: crates/bevy_gltf/src/loader.rs [hunk 2]
```diff
@@ -604,22 +605,23 @@ fn load_node(
         match camera.projection() {
             gltf::camera::Projection::Orthographic(orthographic) => {
                 let xmag = orthographic.xmag();
-                let ymag = orthographic.ymag();
+                let _ymag = orthographic.ymag();
```
Comment (mockersf) [inline crates/bevy_gltf/src/loader.rs:608]:
```suggestion
```

**Review by mockersf — APPROVED** [review]

**Review by superdump — APPROVED** [review]
LGTM. And I saw the discussion in Discord and that it has been tested, exporting a scene with an orthographic camera from current Blender to glTF and it looks correct.

**Pull request closed without merging**